packages/basti/src/common/resource-type.ts

```diff
@@ -11,8 +11,10 @@ export type ManagedResourceType =
 export const TargetTypes = {
   RDS_INSTANCE: 'RDS_INSTANCE',
   RDS_CLUSTER: 'RDS_CLUSTER',
-  ELASTICACHE_CLUSTER: 'ELASTICACHE_CLUSTER',
-  ELASTICACHE_NODE: 'ELASTICACHE_NODE',
+  ELASTICACHE_CLUSTER: 'ELASTICACHE_REDIS_CLUSTER',
```

Comment:
Please, rename the keys in this object to `ELASTICACHE_REDIS_CLUSTER` and `ELASTICACHE_REDIS_NODE`